.pre-commit-config.yaml

```
@@ -28,21 +28,21 @@ repos:
         exclude: .pre-commit-config.yaml|pages/developers/style.md
 
   - repo: https://github.com/codespell-project/codespell
-    rev: v2.2.1
+    rev: v2.2.2
     hooks:
       - id: codespell
         exclude: (^Gemfile\.lock|\.svg)$
         args: ["-L", "hist,sur,nd,theses,ser"]
 
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: "v3.0.0-alpha.0"
+    rev: "v3.0.0-alpha.4"
```

```suggestion
    rev: "v3.0.0-alpha.0"
```

going to try to see if this I can get Henry's suggestion to work again here.